doc/quickstart/install.rst

```diff
@@ -208,3 +208,40 @@ yourself if you are just running diagnostics:
  - DKRZ-Mistral: `esmvaltool` is available on login nodes (`mistral.dkrz.de`) and pre- and post-processing
    nodes (`mistralpp.dkrz.de`) after login and module loading via `module load esmvaltool`; the command
    `module help esmvaltool` provides some information about the module.
+
+Installation from the conda lock file
+-------------------------------------
+
+A fast conda environment creation is possible using the provided conda lock files.
+To build a conda lock file install the `conda-lock` package first:
+
+.. code-block:: bash
+
+   conda install -c conda-forge conda-lock
+
+ then run
```

Comment:
```suggestion
then run
```

Reply:
The space turns this into a code block, so remove the space to keep this as text!